eclipse-scout-core/src/form/Form.ts

```diff
@@ -1278,6 +1291,7 @@ export class Form extends Widget implements FormModel, DisplayParent {
     if (this.$header) {
       this.$title.text(this.title);
       this.$header.toggleClass('no-title', !this.title && !this.subTitle);
+      aria.linkElementWithLabel(this.$container, this.$title);
```

Comment:
This belongs to _renderHeader. _renderTitle is called whenever the title property changes, but the linking can be done initially when this.$title is created